backend/tests/integration/editor/cassettes/test_create_snow_ticket_returns_500_on_error.yaml

```diff
@@ -0,0 +1,104 @@
+      Connection:
+      - keep-alive
+      Content-Type:
+      - application/json
+      User-Agent:
+      - python-requests/2.22.0
+    method: GET
+    uri: https://cerntraining.service-now.com/api/now/table/sys_user_grmember?sysparm_fields=user.name%2Cgroup.name%2Cuser.email%2Cuser.sys_id&sysparm_query=group.nameSTARTSWITHInspire+Information+system
+  response:
+    body:
+      string: '{"error":{"message":"User Not Authenticated","detail":"Required to
+        provide Auth information"},"status":"failure"}'
+    headers:
+      Cache-Control:
+      - no-cache,no-store,must-revalidate,max-age=-1
+      Connection:
+      - keep-alive
+      Content-Type:
+      - application/json;charset=UTF-8
+      Date:
+      - Tue, 31 Jan 2023 13:41:16 GMT
+      Expires:
+      - '0'
+      Pragma:
+      - no-store,no-cache
+      Server:
+      - snow_adc
+      Server-Timing:
+      - sem_wait;dur=1, sesh_wait;dur=0
+      Set-Cookie:
+      - BIGipServerpool_cerntraining=368c7d8307adbe2ec0fcbb5476383cd5; httponly; secure;
+        path=/; SameSite=None
+      - JSESSIONID=FE900BF1C591109F9365DBCA6AA54961; Path=/; HttpOnly; secure; SameSite=None
+      Strict-Transport-Security:
+      - max-age=63072000; includeSubDomains
+      Transfer-Encoding:
+      - chunked
+      WWW-Authenticate:
+      - BEARER realm="Service-now"
+      content-length:
+      - '113'
+    status:
+      code: 401
+      message: Unauthorized
+- request:
+    body: null
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
+      - gzip, deflate
+      Connection:
+      - keep-alive
+      Content-Type:
+      - application/json
+      User-Agent:
+      - python-requests/2.22.0
+    method: GET
+    uri: https://cerntraining.service-now.com/api/now/table/sys_user_grmember?sysparm_fields=user.name%2Cgroup.name%2Cuser.email%2Cuser.sys_id&sysparm_query=group.nameSTARTSWITHInspire+Information+system
+  response:
+    body:
+      string: '{"error":{"message":"User Not Authenticated","detail":"Required to
+        provide Auth information"},"status":"failure"}'
+    headers:
+      Cache-Control:
+      - no-cache,no-store,must-revalidate,max-age=-1
+      Connection:
+      - keep-alive
+      Content-Type:
+      - application/json;charset=UTF-8
+      Date:
+      - Tue, 31 Jan 2023 13:41:16 GMT
+      Expires:
+      - '0'
+      Pragma:
+      - no-store,no-cache
+      Server:
+      - snow_adc
+      Server-Timing:
+      - sem_wait;dur=0, sesh_wait;dur=0
+      Set-Cookie:
+      - BIGipServerpool_cerntraining=368c7d8307adbe2ec0fcbb5476383cd5; httponly; secure;
+        path=/; SameSite=None
+      - JSESSIONID=FCCB470C6F652ECE145BC7150EEDFD7D; Path=/; HttpOnly; secure; SameSite=None
+      Strict-Transport-Security:
+      - max-age=63072000; includeSubDomains
+      Transfer-Encoding:
+      - chunked
+      WWW-Authenticate:
+      - BEARER realm="Service-now"
+      content-length:
+      - '113'
+    status:
+      code: 401
```

Comment:
this doesn't return `500` is it on purpose?

Reply:
no, the cassette is wrong, thanks